DataLayer/Repository/Repository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataLayer.Repository
+{
+    internal class Repository : DataLayer.Repository.IRepository.IRepository
+    {
```

Comment:
What is it needed for?

Reply:
it is data layer, that is what I cloned from dev